src/garage/experiment/local_runner.py

```diff
@@ -146,7 +146,6 @@ def save(self, epoch, paths=None):
 
         # Save states
         params['env'] = self.env
-        params['algo'] = self.algo
```

Comment:
uh is this here intentionally? @naeioi @zequnyu

Reply:
oh.. I removed it by mistake. I will add it back. 

Whether it should stay or not is a separate issue I think.

Reply:
I think it's fine to keep this. There are a few scenarios where one might not want to save the algorithm.